assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -128,21 +165,36 @@ function MyStories() {
         layoutStyle={viewStyle}
         onPress={handleViewStyleBarButtonSelected}
       />
-      <StoryGrid>
-        {filteredStories.map((story) => (
-          <CardGridItem key={story.id}>
-            <CardPreviewContainer
-              onOpenInEditorClick={() => {}}
-              onPreviewClick={() => {}}
-              previewSource={'http://placeimg.com/225/400/nature'}
-            />
-            <CardTitle
-              title={story.title}
-              modifiedDate={story.modified.startOf('day').fromNow()}
-            />
-          </CardGridItem>
-        ))}
-      </StoryGrid>
+      <FontProvider>
+        <TransformProvider>
```

Comment:
Instead of `TransformProvider` we just need to modify `transform/context.js` to provide empty functions. It's really meant as a no-op context by default.